app/game/Swarm/App.hs

```diff
@@ -47,7 +56,7 @@ app eventHandler =
     { appDraw = drawUI
     , appChooseCursor = chooseCursor
     , appHandleEvent = eventHandler
-    , appStartEvent = enablePasteMode
+    , appStartEvent = pure ()
```

Comment:
Why was `enablePasteMode` removed?

Reply:
I moved it to `buildVty` - I don't see why you would not want the paste mode on reinitialize, and we already have a similar mouse option set there.

When does the mode come into play? Currently, we only have a one-line REPL.

Reply:
Ah, it allows us to strip newlines and paste the whole definition into one line.

So it works with the refactor. 🙂